tests/test_api.py

```diff
@@ -85,7 +94,8 @@ def test_read_only(self, client, entities, endpoint, field):
 
     @pytest.mark.parametrize('endpoint,field', [
         ('/participants', 'blah'),
-        ('/samples', 'blah')
+        ('/samples', 'blah'),
+        ('/studies', 'blah'),
     ])
     def test_unknown_field(self, client, entities, endpoint, field):
```

Comment:
I changed this method in #173 to provide an HTTP method so that we could perform this test for both POST and PATCH. We may want to merge #173 before this one and then you can update this test for study